src/Compilers/CSharp/Portable/Binder/DecisionDagBuilder.cs

```diff
@@ -536,6 +536,7 @@ bool IsDerivedType(TypeSymbol possibleDerived, TypeSymbol possibleBase)
                     {
                         Debug.Assert(recursive.HasAnyErrors);
                         tests.Add(new Tests.One(new BoundDagTypeTest(recursive.Syntax, ErrorType(), input, hasErrors: true)));
+                        continue;
```

Comment:
nit: seems like continue skips just a single statement here so 'else' may be preferable